src/main/java/com/egen/ordermanagment/controller/kafka/KafkaController.java

```diff
@@ -0,0 +1,43 @@
+package com.egen.ordermanagment.controller.kafka;
+
+import com.egen.ordermanagment.dto.OrdersDTO;
+import com.egen.ordermanagment.response.Response;
+import com.egen.ordermanagment.response.ResponseMetadata;
+import com.egen.ordermanagment.response.StatusMessage;
+import com.egen.ordermanagment.services.kafka.producer.ProducerServiceImpl;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@Slf4j
+@RequestMapping("/orders")
+public class KafkaController {
```

Comment:
It doesn't have to new file but you can use your orderControllers as well

Reply:
Hi Gagan, Thank you for your feedback I have removed Kafka Controller and used the orderController placeorder method. Please verify.